### DISCUSSION TITLE
Welcome to the CasADi forum

### DISCUSSION THREAD

**Discussion by jgillis** [casadi/casadi#3071] [Announcements]
You may post any discussions and support for using the optimization framework CasADi. Also feature requests are welcome. Always make sure that the topic of the thread is descriptive for your problem! If you have a follow-up question that deviates from the original topic, please start a new thread. Off-topic posts will be deleted.

If you want to include a code snippet which is longer than a few lines, consider using Gist (https://gist.github.com).

If your question is related to a suspected or confirmed bug, please try to isolate the issue as much as possible and create an issue in the issue tracker at Github. Reducing a complex code to a minimal failing example can take hours, and we do not have the resources to do this for users.

If you have found a satisfactory answer consider adding it to the **FAQ** or **Onboarding guide** at our [wiki](https://github.com/casadi/casadi/wiki).

**Comment by aminrbspace** [comment]
Hi all, Is there anybody who works on Nonlinear MPC of quadrotor using Casadi? I have issues in weight metrices tuning to follow the desired trajectory.
